routes/index.js

```diff
@@ -36,20 +38,23 @@ module.exports = function(app) {
         // Here, we just use the host for the application making the request,
         // but you can hard code it or use something different if need be
         var url = 'http://' + request.headers.host + '/outbound';
-        
-        // Place an outbound call to the user, using the TwiML instructions
-        // from the /outbound route
-        client.makeCall({
+
+        const options = {
             to: request.body.phoneNumber,
             from: config.twilioNumber,
-            url: url
-        }, function(err, message) {
-            console.log(err);
-            if (err) {
-                response.status(500).send(err);
+            url: url,
+        };
+
+        // Place an outbound call to the user, using the TwiML instructions
+        // from the /outbound route
+        client.api.v2010.accounts(config.accountSid).calls.create(options)
+          .then((error, message) => {
```

Comment:
The `then` callback does not seem right. You can either pass to `then` 2 callbacks: the first for a fulfilled promise and in this case with a single argument. The second for a rejected promise also with a single Error argument. 
The other option is to pass only a `onFulfilled` callback to `then` and chain afterwards a `catch` method with a single `onReject` callback.

Look at an example of the first option in the Twilio-node specs https://github.com/twilio/twilio-node/blob/next-gen/spec/integration/rest/api/v2010/account/call.spec.js#L29

And of the second in https://github.com/TwilioDevEd/server-notifications-node/blob/next-gen/twilioClient.js